src/App.tsx

```diff
@@ -1,24 +1,86 @@
 import React from 'react';
 import './App.scss';
 
-const App: React.FC = () => {
-  setInterval(() => {
-    const date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
+type State = {
+  isClockVisible: boolean;
+  clockName: number | string;
+  date: string;
 };
 
+class App extends React.Component<{}, State> {
+  state: State = {
+    isClockVisible: true,
+    clockName: 'New',
+    date: (new Date()).toLocaleTimeString(),
+  };
+
+  componentDidMount() {
+    setInterval(() => {
```

Comment:
Let's create the component `Clock` and handle all about the time inside it.
Also, you should clear the interval when the component disappears from the page (componentWillUnmount)